platform-configs/minnowboard-turbot.robot

```diff
@@ -3,36 +3,82 @@ Resource    include/default.robot
-${FLASH_SIZE}=                          ${8*1024*1024}
-${FLASH_LENGTH}=                        ${EMPTY}
-${TIANOCORE_STRING}=                    to enter Boot Manager Menu
-${BOOT_MENU_STRING}=                    Please select boot device:
-${BOOT_MENU_KEY}=                       ${F7}
-${SETUP_MENU_KEY}=                      ${F2}
-${SETUP_MENU_STRING}=                   Select Entry
-${EDK2_IPXE_CHECKPOINT}=                iPXE Shell
-${MANUFACTURER}=                        MinnowBoard
-${CPU}=                                 Intel Atom E3845 SoC
-${POWER_CTRL}=                          RteCtrl
-${FLASH_VERIFY_METHOD}=                 tianocore-shell
-${FLASH_VERIFY_OPTION}=                 UEFI Shell
-${WIFI_CARD}=                           ${EMPTY}
-${MAX_CPU_TEMP}=                        ${EMPTY}
-
-${DEVICE_USB_KEYBOARD}=                 Logitech, Inc. Keyboard K120
+${INITIAL_DUT_CONNECTION_METHOD}=           Telnet
+${DUT_CONNECTION_METHOD}=                   ${INITIAL_DUT_CONNECTION_METHOD}
+${PAYLOAD}=                                 tianocore
+${RTE_S2_N_PORT}=                           13542
+${FLASH_SIZE}=                              ${8*1024*1024}
+${FLASH_LENGTH}=                            ${EMPTY}
+${TIANOCORE_STRING}=                        to enter Boot Manager Menu
+${BOOT_MENU_STRING}=                        Please select boot device:
+${BOOT_MENU_KEY}=                           ${F7}
+${SETUP_MENU_KEY}=                          ${F2}
+${SETUP_MENU_STRING}=                       Select Entry
+${EDK2_IPXE_CHECKPOINT}=                    iPXE Shell
+${MANUFACTURER}=                            MinnowBoard
+${CPU}=                                     Intel Atom E3845 SoC
+${POWER_CTRL}=                              RteCtrl
+${FLASH_VERIFY_METHOD}=                     tianocore-shell
+${FLASH_VERIFY_OPTION}=                     UEFI Shell
+${WIFI_CARD}=                               ${EMPTY}
+${MAX_CPU_TEMP}=                            ${EMPTY}
+
+${DEVICE_USB_KEYBOARD}=                     Logitech, Inc. Keyboard K120
 
 # Supported test environments
-${TESTS_IN_FIRMWARE_SUPPORT}=           ${TRUE}
-${TESTS_IN_UBUNTU_SUPPORT}=             ${FALSE}
-${TESTS_IN_DEBIAN_SUPPORT}=             ${FALSE}
-${TESTS_IN_WINDOWS_SUPPORT}=            ${FALSE}
+${TESTS_IN_FIRMWARE_SUPPORT}=               ${TRUE}
+${TESTS_IN_UBUNTU_SUPPORT}=                 ${TRUE}
+${TESTS_IN_DEBIAN_SUPPORT}=                 ${FALSE}
+${TESTS_IN_WINDOWS_SUPPORT}=                ${FALSE}
+${UEFI_COMPATIBLE_INTERFACE_SUPPORT}=       ${TRUE}
 
 # Regression test flags
-${CUSTOM_BOOT_MENU_KEY_SUPPORT}=        ${TRUE}
+${CUSTOM_BOOT_MENU_KEY_SUPPORT}=            ${TRUE}
+${SERIAL_NUMBER_VERIFICATION}=              ${TRUE}
+${FIRMWARE_NUMBER_VERIFICATION}=            ${TRUE}
+${PRODUCT_NAME_VERIFICATION}=               ${TRUE}
+${UEFI_SHELL_SUPPORT}=                      ${TRUE}
+${EXTERNAL_HDMI_DISPLAY_SUPPORT}=           ${FALSE}
```

Comment:
There is a HDMI on this board, so we should test it

Reply:
Fixed in [this commit](https://github.com/Dasharo/open-source-firmware-validation/pull/441/commits/0b52c12d8825278cea0abf2e2b9dc6aa2a5ca401)

Reply:
yes, commit fixed, but still it is FALSE, why? Unresolving